EIPS/EIP-X-add-token-uri-kylanhurt.md

```diff
@@ -0,0 +1,60 @@
+## Preamble
```

Comment:
Please update the formatting as per the new formatting standards in EIP-1 and EIP-X.